src/components/SystemManagedList.js

```diff
@@ -336,5 +344,5 @@ export default class SystemManagedList extends Component {
     this.getCurations(curationsToGet)
   }
 
-  updateList() {}
+  updateList(_) {}
```

Comment:
I'm not so sure about this change, what the `_` is supposed to do?

Reply:
This fixes a linting issue saying that the `updateList()` function doesn't require any arguments because this function is called with one argument. See line `339` for example.

If I remove the `_` variable argument, I get this:
<img width="743" alt="SystemManagedList_js_—_website" src="https://user-images.githubusercontent.com/899175/54235727-9737e600-451a-11e9-8cea-a2478c5f3a66.png">